sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -117,6 +117,7 @@ def _from_generated(cls, generated):
             size=generated.size,
             tags=generated.tags,
             content_permissions=generated.writeable_properties,
+            repository_name=kwargs.get("repository_name"),
```

Comment:
in line 101, you have `kwargs.get("repository_name", None)`. Will the generated object always have `repository_name`, so you don't have to return None otherwise?

Reply:
It should always have `repository_name` because I have to add that additionally to the returned object, however it's good practice to include the `None` default so I will add that.